ads/_ping_.js

```diff
@@ -18,10 +18,13 @@ import {validateData} from '../3p/3p';
 import {dev} from '../src/log';
 
 /**
+ * A fake ad network integration that is mainly used for testing
+ * and demo purpose.
  * @param {!Window} global
  * @param {!Object} data
  */
 export function _ping_(global, data) {
+  global.dataAsParam = data; // for testing only. see #10628
```

Comment:
Can you rename to `networkIntegrationDataParamForTesting`?

I know it's a mouthful, but this particular thing is really weird and I don't want future me to get confused :)